src/main/java/com/revature/chronicle/ChronicleApplication.java

```diff
@@ -14,8 +20,12 @@ public static void main(String[] args) {
 		SpringApplication.run(ChronicleApplication.class, args);
 	}
 
-	@GetMapping("/hello")
+	@CrossOrigin
+	@GetMapping(path = "/hello", produces = MediaType.APPLICATION_JSON_VALUE)
 	public String hello(@RequestParam(value = "name", defaultValue = "World") String name) {
```

Comment:
Is this needed?

Reply:
Not necessarily, We left it in there so people could see how to interact with the backend, it can be removed

Reply:
Would you like it to be removed?

Reply:
From the main application, yes.
If it is necessary or desired to be included for development, then create a separate `TestController.java` and only include it under the `local` (or however you wish to name that profile) profile.

Reply:
Removed